engine/class_modules/sc_shaman.cpp

```diff
@@ -5865,6 +5865,17 @@ struct chain_lightning_t : public chained_base_t
   {
     chained_base_t::execute();
 
+    if ( exec_type == spell_variant::NORMAL && p()->specialization() == SHAMAN_ELEMENTAL )
```

Comment:
I'm not sure if we have any sources of background Chain Lightnings today actually, but feels better safe than sorry to start scoping foreground only effects in a check for that.